packages/liveblocks-react/src/room.tsx

```diff
@@ -287,10 +287,15 @@ function makeExtrasForClient<M extends BaseMetadata>(client: OpaqueClient) {
       if (room === null) return;
 
       // Retrieve threads that have been updated/deleted since the last requestedAt value
-      requests.push(store.getThreadsUpdates(room.id));
+      requests.push(store.fetchThreadsDeltaUpdate(room.id));
     });
 
-    await Promise.allSettled(requests);
+    try {
+      await Promise.allSettled(requests);
+    } catch (err) {
+      // When polling, we don't want to throw errors, ever
+      console.warn(`Polling new threads failed: ${String(err)}`);
+    }
```

Comment:
This try/catch won't be needed, because the promise returned by `Promise.allSettled()` will never be rejected and will always succeed. (Different from `Promise.all()`.)